.devenv.hosting.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
       container_name: s01
       config_path: ../neofs-dev-env/services/storage/.storage.env
       wallet_path: ../neofs-dev-env/services/storage/wallet01.json
-      local_config_path: ./TemporaryDir/empty-password.yml
+      local_config_path: ./ConfigsDir/empty-password.yml
```

Comment:
Do we really need these? IIUC it's about wallet unlock files with known passwords, nothing really special/unknown here.